snuba/migrations/snuba_migrations/profiles/0003_add_device_architecture.py

```diff
@@ -0,0 +1,38 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.PROFILES,
+                table_name="profiles_local",
+                column=Column(
+                    "architecture",
+                    String(Modifiers(low_cardinality=True, default="'unknown'")),
+                ),
+                after="device_os_version",
+            )
+        ]
+
+    def backwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.DropColumn(
+                storage_set=StorageSetKey.PROFILES,
+                table_name="profiles_local",
+                column_name="architecture",
+            )
+        ]
+
+    def forwards_dist(self) -> Sequence[operations.SqlOperation]:
+        return []
+
+    def backwards_dist(self) -> Sequence[operations.SqlOperation]:
+        return []
```

Comment:
Is there a reason why the column isn't added to the distributed tables? In production, all queries go through the distributed tables. If you don't have the column defined in the distributed table you won't be able to access the column in the local tables

Reply:
@nikhars I was looking at a previous migration ([002](https://github.com/getsentry/snuba/blob/master/snuba/migrations/snuba_migrations/profiles/0002_disable_vertical_merge_algorithm.py)) as a reference.

Thanks for pointing that out.